src/main/java/bio/terra/common/logging/LoggingUtils.java

```diff
@@ -119,8 +118,7 @@ protected static void initializeLogging(ConfigurableApplicationContext applicati
         // seems to work. But if we encounter future issues in human-readable logging control, this
         // is a reasonable place to look more closely.
         logbackLogger.detachAndStopAllAppenders();
-        new ContextInitializer(logbackLogger.getLoggerContext())
-            .configureByResource(ResourceUtils.getURL("classpath:logback.xml"));
+        new ContextInitializer(logbackLogger.getLoggerContext()).autoConfig();
```

Comment:
I think this is the same thing. The old method was deprecrated, and said:
```
    /**
     * This method is deprecated with no replacement
     * @param url
     * @throws JoranException
     */
    @Deprecated
    public void configureByResource(URL url) throws JoranException {
```

https://github.com/qos-ch/logback/blob/v_1.4.8/logback-classic/src/main/java/ch/qos/logback/classic/util/ContextInitializer.java#L55-L61